src/main/java/net/openhft/chronicle/core/values/LongValue.java

```diff
@@ -19,7 +19,11 @@
 package net.openhft.chronicle.core.values;
 
 import net.openhft.chronicle.core.Jvm;
+import net.openhft.chronicle.core.internal.threads.VanillaThreadLock;
 import net.openhft.chronicle.core.io.Closeable;
+import net.openhft.chronicle.core.threads.ThreadLock;
+
+import java.util.concurrent.TimeUnit;
 
 public interface LongValue extends Closeable {
```

Comment:
The relation between a LongValue and a ThreadLock is unclear. Why are they related? What does the LongValue reflect etc.